Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -91,59 +93,141 @@ public void onResult(@Nullable Configuration configuration, @Nullable Exception
                     return;
                 }
 
-                sharedPrefsWriter.persistVenmoVaultOption(activity, request.getShouldVault() && braintreeClient.getAuthorization() instanceof ClientToken);
+                if (request.getPaymentMethodUsage() == VenmoPaymentMethodUsage.UNSPECIFIED) {
+                    braintreeClient.sendAnalyticsEvent("pay-with-venmo.app-switch.failed");
+                    callback.onResult(new BraintreeException("Payment method usage must be set."));
```

Comment:
Is there any reason that we're throwing an exception instead of just requiring this to be passed in when constructing a `VenmoRequest`? It seems like that would allow us to remove the `UNSPECIFIED` case from the enum and maybe alleviate some merchant confusion.

Reply:
I think this was the requested behavior on the Android side - possibly to align the requests with iOS? But I agree it would make it easier on merchants to require `paymentMethodUsage` in the request. @demerino Is `UNSPECIFIED` still required here or can we simplify this?

Reply:
I'm fairly certain this is a case where it's OK for iOS and Android to differ! We can add a NEXT_MAJOR_VERSION note on iOS to align with Android later on.

Reply:
Went ahead and removed `UNSPECIFIED`!